.github/workflows/pr-gpu.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
         gpu_num: ${{ matrix.gpu_num }}
         git_repo: databricks/megablocks
         pip_deps: "[all,testing]"
-        pytest_command: "coverage run -m pytest tests" # todo: remove tests from pytest tests when we delete all tests outside of MegaBlocks repo
+        pytest_command: "coverage run -m pytest tests"  # todo: remove `tests` when delete tests outside megablocks dir
```

Comment:
why this change?

Reply:
My initial comment was too long and the yaml linter complained.

Reply:
I just fixed the comment and made it clearer. LMK if it is good now.

Reply:
put on a new line.